shtdlib.sh

```diff
@@ -1654,6 +1654,17 @@ function ln_sf {
     debug 10 "Successfully created symlink"
 }
 
+# Create string of random characters
+#  - First param is length, default: 20
+#  - Second param is characters, default: A-Za-z0-9_
```

Comment:
I'd clarify that dash is not one of the default characters.

Reply:
Done